tests/python_functional/src/syslog_ng_config/syslog_ng_config.py

```diff
@@ -65,10 +65,10 @@ def write_config(self, config_path):
     def set_version(self, version):
         self.__syslog_ng_config["version"] = version
 
-    def add_include(self, include):
+    def set_include(self, include):
```

Comment:
I liked `add_include()`, as the function itself `append`s, not `update`s.

Reply:
itHere I wanted some consistency in function names. My plan was the following:
- every function which returns with some object will started with "create_" prefix (it was already ok)
- other functions should stars with an other prefix (to avoid confusion: one is return with something other is not). Started thinking with "global_options()", and I concluded that the clearer prefix is "set_"